executor/insert.go

```diff
@@ -385,7 +385,7 @@ func (e *InsertExec) Open(ctx context.Context) error {
 // decodeOldHandle decode old handle by key-value pair.
 // The key-value pair should only be a table record or a distinct index record.
 // If the key is a record key, decode handle from the key, else decode handle from the value.
-func decodeOldHandle(k keyWithDupError, value []byte) (oldHandle int64, err error) {
+func (e *InsertExec) decodeOldHandle(k keyWithDupError, value []byte) (oldHandle int64, err error) {
```

Comment:
Could we rename it to decodeHandle? From the code, I can not tell why call it decodeOldHandle.